scripts/build-changelog.php

```diff
@@ -0,0 +1,144 @@
+<?php
+
+/*
+* This script generates changelog based on the json files in "../.changes/nextrelease/"
+* and generates a new TAG for the build.
+*/
+
+$dir = "../.changes/";
+$newServiceFlag = False;
+
+
+function readChangelog($dir){
+	$dir = $GLOBALS['dir']."/nextrelease/";
+	$CHANGELOG=[];
+	if (is_dir($dir)){
```

Comment:
Are we going to remove 'nextrelease' directory sometime? When can it not be a dir?
If that's the case, could we throw exception let release to stop instead of passing silently? (Maybe you are thinking doc only releases)?